examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/LinearRegressionFromRestoredPythonModel.scala

```diff
@@ -0,0 +1,123 @@
+package org.platanios.tensorflow.examples.python2scala
+
+import java.io._
+import java.nio.file.Paths
+
+import com.typesafe.scalalogging.Logger
+import org.platanios.tensorflow.api.core.Graph
+import org.platanios.tensorflow.api.ops.Output
+import org.platanios.tensorflow.api.{Op, Session, Shape, Tensor, tf}
+import org.slf4j.LoggerFactory
+import org.tensorflow.framework.MetaGraphDef
+
+import scala.collection.mutable.ArrayBuffer
+import scala.util.Random
+
+/**
+  * @author Luca Tagliabue
+  *         Purpose: take a model trained with the Python API of Tensor Flow and restore
+  *         it inside a Scala environment in order to continue the training process. The
+  *         model was saved using tf.train.Saver()
+  *
+  *         Version used:
+  *         TensorFlow version: 1.8.0rc1
+  *         ScalaTensorFlow version: 0.2.0-SNAPSHOT
+  */
+object LinearRegressionFromRestoredPythonModel {
+  private[this] val logger = Logger(LoggerFactory.getLogger("Examples / Linear Regression"))
+  private[this] val random = new Random(22)
+  private[this] val weight = random.nextFloat()
+
+  def main(args: Array[String]): Unit = {
+    val meta = "examples/src/main/scala/org/platanios/tensorflow/examples/python2scala/virgin-linear-regression-pull-request.meta"
```

Comment:
Can you move those files to a resources directory? E.g., "examples/src/main/resources". Then you don't need to hardcode the path this way and you can find the paths directly using the Java resource manager. This will also allow for your example to work if packaged and distributed in a JAR file.

Reply:
This is actually my main request before I merge.